<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="browser_layout">
     <item>
      <widget class="QWebView" name="browser_site" native="true">
       <property name="url" stdset="0">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWebView" name="browser_translate" native="true">
       <property name="url" stdset="0">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="pixmap">
         <pixmap resource="../../../image/image.qrc">:/english_table.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../image/image.qrc"/>
 </resources>
 <connections/>
</ui>
